vars/Constants.groovy

```diff
@@ -1,14 +1,16 @@
 class Constants {
+    static final String USERNAME = "jenkins";
     static final String[] IGNORE_LABELS = ['.ignore'];
-    static final String REMOTE_FS = "/home/jenkins";
-    static final String WIN_REMOTE_FS = "C:\\Users\\jenkins";
+    static final String REMOTE_FS = "/home/${USERNAME}";
+    static final String WIN_REMOTE_FS = "C:\\Users\\${USERNAME}";
+    static final String OSX_REMOTE_FS = "/Users/${USERNAME}";
     
     // This the key that'll be used for SSHLauncher in CreateNewNode
     static final String SSH_CREDENTIAL_ID = "";
     
     static final String SLAVE_JAR_LOCATION = "<jenkins_URL/>/jnlpJars/slave.jar";
     static final String WGET_SLAVE_JAR = "\"wget -q --no-check-certificate -O slave.jar ${SLAVE_JAR_LOCATION} ; java -jar slave.jar\"";
-    static final String SSH_COMMAND = "ssh -C -i ${SSH_KEY_LOCATION} <userName/>@";
+    static final String SSH_COMMAND = "ssh -C -i ${SSH_KEY_LOCATION} ${USERNAME}@";
     static final String SSH_KEY_LOCATION = "";
 
     static final String JENKINS_URL = "https://ci.adoptopenjdk.net/";
```

Comment:
Jenkins has a variable[1] for this already, can we remove our definition and update the `SLAVE_JAR_LOCATION` to use it?

[1] https://wiki.jenkins.io/display/JENKINS/Building+a+software+project (`JENKINS_URL`).

Other than that, this lgtm, please update the PR title and description and squash into 1 commit.